Decisions.MongoDB/ReplaceBulkDocumentStep.cs

```diff
@@ -0,0 +1,92 @@
+using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Linq;
+using System.Reflection;
+using DecisionsFramework;
+using DecisionsFramework.Design.Flow;
+using DecisionsFramework.Design.Flow.Mapping;
+using DecisionsFramework.Utilities;
+using MongoDB.Driver;
+
+namespace Decisions.MongoDB
+{
+    public class ReplaceBulkDocumentStep : BaseReplaceStep
```

Comment:
Let's just get rid of this since there's no obvious method in Mongo's API to use.